firefox_puppeteer/tests/test_utils.py

```diff
@@ -0,0 +1,74 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
+
+
+class TestUtils(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        # Clear all previous history and cookies.
+        self.places.remove_all_history()
+        self.marionette.delete_all_cookies()
+
+        # The test urls.
+        self.urls = [
```

Comment:
Lets not assign it to `self.urls` yet but to a local variable. nit: no need for the comment in line 17.